docs/cli/start.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "wash start"
+draft: false
+sidebar_position: 23
+description: "wash start command reference"
+--- 
+
+`wash start` assists in launching an actor or a provider from an OCI reference on a host. The user may specify if they want to start an actor or a provider and pass the OCI reference of that entity to the command. By default, the actor or provider are auctioned in the lattice for a suitable host. To specify a host to launch the actor or provider, pass the host ID using the `--host-id` flag. Actors or providers can also be started from local files using the prefix `file://` followed by the file path. By default, loading from files is permitted and can be disabled using the `WASMCLOUD_ALLOW_FILE_LOAD` environment variable. Following are the available subcommads:
```

Comment:
One more tiny change: The default is actually to not allow file loading on the host. However, for `wash up` (since it is mostly using for development), it is the default. It would be good to clarify here